### DISCUSSION TITLE
after upgrade to latest piler "Message Failed Verification" on every message

### DISCUSSION THREAD

**Discussion by ckruijntjens** [jsuto/piler#212] [General]
Hi,

After the upgrade i got on every message the error "Message Failed Verification". What can be wrong and where to start with troubleshoot.

![Screenshot_2024-12-20_10-20-16](https://github.com/user-attachments/assets/704bf0ee-146c-4734-9b07-de15714c9e69)

**Comment by jsuto** [comment]
The mail logs. Look for lines with pilerget.

**Reply by ckruijntjens** [reply]
> The mail logs. Look for lines with pilerget.

After the upgrade suddenly piler can not connect to mariadb.

2024-12-20T15:39:12.057563+01:00 piler piler[2400]: ERROR: cant connect to mysql server
2024-12-20T15:39:12.057758+01:00 piler piler[2400]: cannot connect to mysql server

**Reply by ckruijntjens** [reply]
> The mail logs. Look for lines with pilerget.

I can connect with the username and password on the command line (on the piler database)

what can be the issue here? Do i need some sort of package with the new mailpiler version?

**Comment by ckruijntjens** [comment]
when doing a ./configure i get error: 

configure: WARNING: unrecognized options: --with-database=mysql

**Comment by ckruijntjens** [comment]
I did a full reinstall and now its working. except with mailpiler 1.4.7 now the script to authenticate to mailcow is not working anymore.

i get an 500 error. local accounts are working. but thats a other issue.
